app/controllers/autocomplete_controller.rb

```diff
@@ -138,7 +138,7 @@ def external_work
 
   # encodings for importing
   def encoding
-    encodings = Encoding.name_list.select {|e| e.match(/#{params[:term]}/i)}
+    encodings = Encoding.name_list.select {|e| e.match(/#{ Regexp.escape(params[:term]) }/i)}
```

Comment:
If you want to fix the Hound violations, I'd suggest adjusting the spacing to be:
```suggestion
    encodings = Encoding.name_list.select { |e| e.match(/#{Regexp.escape(params[:term])}/i) }
```